package.json

```diff
@@ -30,3 +30,12 @@
     "sync-request": "^3.0.1"
   }
 }
+# Currently after running npm install we get the following warnings:
```

Comment:
This wasn't suppose to be like this. You have to google about those `npm WARN` that come because we need to add info in package.json, and add the corresponding info in the fields in the contents of package.json. 

.json is a format and the content you added doesn't fit in that format